src/Reflex/Process.hs

```diff
@@ -34,10 +39,15 @@ data ProcessConfig t i = ProcessConfig
   -- ^ "stdin" input to be fed to the process
   , _processConfig_signal :: Event t P.Signal
   -- ^ Signals to send to the process
+  , _processConfig_createProcessMethod
```

Comment:
You can just call this `_processConfig_createProcess`

Reply:
Done, thanks!